src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -73,6 +75,14 @@ public class KNNConstants {
 
     // Lucene specific constants
     public static final String LUCENE_NAME = "lucene";
+    public static final String LUCENE_SQ_CONFIDENCE_INTERVAL = "confidence_interval";
+    public static final int DYNAMIC_CONFIDENCE_INTERVAL = 0;
+    public static final double MINIMUM_CONFIDENCE_INTERVAL = 0.9;
+    public static final double MAXIMUM_CONFIDENCE_INTERVAL = 1.0;
+    public static final String LUCENE_SQ_BITS = "bits";
+    public static final Integer LUCENE_SQ_DEFAULT_BITS = 7;
+    public static final List<Integer> LUCENE_SQ_BITS_SUPPORTED = List.of(7);
```

Comment:
if the constant is getting used once, its better to move the constant in that class only rather than putting it in a generic constants class.